examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -356,7 +357,7 @@ func copyableType(t *types.Type) bool {
 		return false
 	}
 	// TODO: Consider generating functions for other kinds too.
-	if t.Kind != types.Struct {
+	if t.Kind != types.Struct && t.Kind != types.Alias {
```

Comment:
I really need you to explain what this is doing and why?  Now we deepcopy other types beside struct?  I'm OK with that but why Alias?

Reply:
You should probably be doing the lookup here to ensure the alias is to a struct.